holonote/app/tabulator.py

```diff
@@ -61,6 +64,7 @@ def new_style(row):
             buttons={"delete": '<i class="fa fa-trash"></i>'},
             show_index=False,
             selectable=True,
+            **self.tabulator_kwargs,
```

Comment:
If it's a `param` should this be wrapped in a function decorated with `pn.depends('tabulator_kwargs')`, running `self.tabulator.param.update(**tabulator_kwargs)`?

Reply:
I'm guessing not, since `tabulator_kwargs` (and _create_tabulator) are intended to only be used once. But it's a good question for someone with a greater understanding to jump in here

Reply:
I agree that the `tabulator_kwargs` should be synced with the `Tabulator`. There actually is an API on `Reactive` objects to do this, specifically, could you try:

```suggestion
            refs=self.param.tabulator_kwargs
```

This will update keep the `tabulator_kwargs` dictionary synced with the `Tabulator` parameters.

Reply:
So my understanding (which I just pushed changes for) is that it is appropriate to make `tabulator_kwargs` into a param (I'm guessing now requiring `allow_refs=True`?), and that it will be synced solely by using the `refs=` API, without the dependence on the `depends` decorator. 

Just curious, what is the reasoning that `tabulator_kwargs` should remain synced with Tabulator beyond initialization? Is it just the general design inclination to allow for future flexibility in case some property about the Tabulator will be internally modified?

Also, am I correct in assuming the `allow_refs=True` is now required or am I confusing concepts? 

Thanks for the guidance!

Reply:
also, if my recent changes look good, please approve and merge! thank you